bot/exts/help_channels/_cog.py

```diff
@@ -178,8 +180,23 @@ async def get_available_candidate(self) -> discord.TextChannel:
                 if last_notification:
                     self.last_notification = last_notification
                     self.bot.stats.incr("help.out_of_channel_alerts")
-
                 channel = await self.wait_for_dormant_channel()
+        message = await channel.fetch_message(channel.last_message_id)
```

Comment:
When there's no available channels message, couldn't this inadvertently fetch the message contain the help instructions?

Reply:
That will raise Forbidden on edit.